### ISSUE TITLE
Please check the formatting of the project readme fiel

### ISSUE THREAD

**Issue by pmario** [tiddly-gittly/watch-fs#4]
@linonetwo ... Please check the formatting on the project readme page: https://github.com/tiddly-gittly/watch-fs ... 

Are you sure, there should be that many H1 elements which are strange links

**Comment by linonetwo** [comment]
Oh, it is just tid, that's why I want https://github.com/Jermolene/TiddlyWiki5/discussions/6590

For now, this is the only choice. Use readme.tid, then there is no rendering, use readme.tid but fake as readme.md, it get at lease some rendering.

**Comment by pmario** [comment]
See: https://tiddlywiki.com/#Using%20TiddlyWiki%20for%20GitHub%20project%20documentation

There is a command eg: `tiddlywiki --verbose --load mywiki.html --rendertiddler ReadMe ./readme.html` that lets you render a ReadMe tiddler and save is as an html tiddler, which can be used to be shown at gh-pages.

TW uses this for https://github.com/Jermolene/TiddlyWiki5  ... The readme there is auto generated. 

the `--output full/path` command can be used to automatically create the file in the right directory of the repo.

**Comment by linonetwo** [comment]
That is good, let me see if I can run it in a github action and do that automatically. Is Jeremy generate readme.md manually?

**Comment by pmario** [comment]
> Is Jeremy generate readme.md manually?

I don't think so. IMO it's part of one of the deploy `*.sh` files. So I lost track about that file. Sorry.
